cmd/cue/cmd/get_go.go

```diff
@@ -549,11 +591,13 @@ func (e *extractor) extractPkg(root string, p *packages.Package) error {
 		}
 	}
 
-	for path := range e.usedPkgs {
-		if !e.done[path] {
-			e.done[path] = true
-			if err := e.extractPkg(root, e.allPkgs[path]); err != nil {
-				return err
+	if !flagOutFile.IsSet(e.cmd) {
+		for pkgPath := range e.usedPkgs {
+			if !e.done[pkgPath] {
+				e.done[pkgPath] = true
+				if err := e.extractPkg(root, e.allPkgs[pkgPath]); err != nil {
+					return err
+				}
```

Comment:
```suggestion
	for pkgPath := range e.usedPkgs {
		if !flagOutFile.IsSet(e.cmd) && !e.done[pkgPath] {
			e.done[pkgPath] = true
			if err := e.extractPkg(root, e.allPkgs[pkgPath]); err != nil {
				return err
```

Reply:
there's a negligible performance difference either way, so there's no need to add a level of nesting here